api/config/cors.php

```diff
@@ -15,7 +15,7 @@
     |
     */
 
-    'paths' => ['api/*', 'sanctum/csrf-cookie'],
+    'paths' => ['api/*', 'sanctum/csrf-cookie', 'graphql'],
```

Comment:
Also we'll need to add admin/graphql.  Any others?  :thinking:

Reply:
Thanks for catching that. Added in [add protected endpoint to cors](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10611/commits/63ad319cbf7d166e70d025f497dea3b3f660eb38)